tests/DomCrawler/CrawlerTest.php

```diff
@@ -385,6 +385,15 @@ public function testHtmlDefault(callable $clientFactory): void
         $this->assertSame('default', $crawler->filter('header')->html('default'));
     }
 
+    /**
+     * @dataProvider clientFactoryProvider
+     */
+    public function testEmptyHtml(callable $clientFactory): void
+    {
+        $crawler = $this->request($clientFactory, '/basic.html');
```

Comment:
Could you also test with no arguments passed to `html()`?

Reply:
@dunglas Done ;)